src/web/app/components/feedback-path-panel/feedback-path-panel.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="background-color-light-green margin-top-15px padding-15px">
+  <div>
+    <b class="feedback-path-title">Feedback Path</b> (Who is giving feedback about whom?)
+  </div>
+  <div ngbDropdown #mainMenu="ngbDropdown" class="margin-top-15px" autoClose="outside">
+    <button id="btn-feedback-path" class="btn btn-light white-space-normal" ngbDropdownToggle [disabled]="!model.isEditable || commonFeedbackPaths.size === 1">
+      <span *ngIf="!model.isUsingOtherFeedbackPath">
+            {{ model.giverType | giverTypeDescription }} will give feedback on <i class="fas fa-arrow-right"></i> {{ model.recipientType | recipientTypeDescription }}
```

Comment:
Nitpicking again, but weird indentation here

Reply:
fixed in f4952ca